### ISSUE TITLE
ENOENT import-map.json

### ISSUE THREAD

**Issue by bernardoadc** [jakedeichert/svelvet#46]
On `npm run dev` first run:

```
Building web_modules with snowpack...

Failed to transform with babel: dist/App.js
Error: unknown: ENOENT: no such file or directory, open '\dist\web_modules\import-map.json'
    at Object.openSync (fs.js:451:3)
    ...

Watching for files...
Server running on http://localhost:8080
```

browser's console:

```
Uncaught TypeError: Failed to resolve module specifier "svelte/internal". Relative references must start with either "/", "./", or "../".
```

(also, for windows, script "clean" should be "rmdir dist /S/Q")

**Comment by jakedeichert** [comment]
What version of svelvet are you on?

> (also, for windows, script "clean" should be "rmdir dist /S/Q")

Also, where are you seeing this? `master` of this branch should not have a `clean` script anywhere.

**Comment by bernardoadc** [comment]
To be honest, I just realized I used YogliB's fork, that's where I got the "clean" script. Heading back to work I'll try using this repo and see if the errors persist

**Comment by jakedeichert** [comment]
I'll close this for now! Please reopen if it still doesn't work 👍

**Comment by bernardoadc** [comment]
Ok, so the problem is still here. Started a fresh project, from Get Started. No dependencies (before I added some libs to see snowpack in action, now I didn't). Full error log:

```
> svelvet

Copied asset src/index.html
Svelte compiled src/App.js
Svelte compiled src/App.js

Building web_modules with snowpack...
[SCAN ERROR]: No files matching "dist/**/*"
⠼ snowpack installing...
× Nothing to install.


Failed to transform with babel: src/App.js
Error: unknown: ENOENT: no such file or directory, open '\dist\web_modules\import-map.json'
    at Object.openSync (fs.js:451:3)
    at Object.readFileSync (fs.js:353:35)
    at readImportMapFile (\node_modules\snowpack\assets\babel-plugin.js:7:15)
    at getImportMap (\node_modules\snowpack\assets\babel-plugin.js:27:24)
    at PluginPass.pre (\node_modules\snowpack\assets\babel-plugin.js:83:28)
    at transformFile (\node_modules\@babel\core\lib\transformation\index.js:96:27)
    at transformFile.next (<anonymous>)
    at run (\node_modules\@babel\core\lib\transformation\index.js:35:12)
    at run.next (<anonymous>)
    at Object.transform (\node_modules\@babel\core\lib\transform.js:27:41) {
  errno: -4058,
  syscall: 'open',
  code: 'ENOENT',
  path: '\\dist\\web_modules\\import-map.json'
}


Failed to transform with babel: src/App.js
Error: unknown: ENOENT: no such file or directory, open '\dist\web_modules\import-map.json'
    at Object.openSync (fs.js:451:3)
    at Object.readFileSync (fs.js:353:35)
    at readImportMapFile (\node_modules\snowpack\assets\babel-plugin.js:7:15)
    at getImportMap (\node_modules\snowpack\assets\babel-plugin.js:27:24)
    at PluginPass.pre (\node_modules\snowpack\assets\babel-plugin.js:83:28)
    at transformFile (\node_modules\@babel\core\lib\transformation\index.js:96:27)
    at transformFile.next (<anonymous>)
    at run (\node_modules\@babel\core\lib\transformation\index.js:35:12)
    at run.next (<anonymous>)
    at Object.transform (\node_modules\@babel\core\lib\transform.js:27:41) {
  errno: -4058,
  syscall: 'open',
  code: 'ENOENT',
  path: '\\dist\\web_modules\\import-map.json'
}

Watching for files...
(node:6364) UnhandledPromiseRejectionWarning: Error: ENOENT: no such
file or directory, watch '\dist'
    at FSWatcher.start (internal/fs/watchers.js:165:26)
    at Object.watch (fs.js:1340:11)
    at watch (\node_modules\servor\servor.js:18:24)
    at module.exports (\node_modules\servor\servor.js:137:5)    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async startDevServer (\node_modules\svelvet\lib\index.js:208:21)
(node:6364) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not
handled with .catch(). (rejection id: 3)
(node:6364) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by jakedeichert** [comment]
What version of svelvet are you on? I can poke at this after work today, but a similar Windows bug was fixed a few days ago and I'm wondering why it's not working now 🤔

**Comment by bernardoadc** [comment]
This: 

```
  "dependencies": {
    "svelte": "^3.18.2"
  },
  "devDependencies": {
    "svelvet": "^0.3.2"
  }
```

It seems a problem with snowpack. `dist` folder was not created at all, nor `dist/web_modules`. Should it seek `import-map.json` if snowpack didn't install anything?

Later on will try on a Mac too, same code.

**Comment by jakedeichert** [comment]
cc @moitias, is the latest svelvet working for you on Windows? Just wondering since there was another Windows issue we fixed and I'm unsure if everything is working completely :)

**Comment by jakedeichert** [comment]
I'll aim to release this fix tomorrow 👍

**Comment by jakedeichert** [comment]
Released as [v0.3.5](https://github.com/jakedeichert/svelvet/releases/tag/v0.3.5)

**Comment by bernardoadc** [comment]
I can assert that v0.3.5 is working on Windows  : > )

I've said I would test in a Mac but could do just now. Bottom line, it was failing as well but for another reason - spaces in my project's folder (error in snowpack saying MODULE_NOT_FOUND, but only half of folder path). Gladly, v0.3.5 also fixed this  : > D
